runtime/manta/Cargo.toml

```diff
@@ -74,6 +74,7 @@ pallet-xcm = { git = 'https://github.com/paritytech/polkadot.git', default-featu
 
 # Self dependencies
 manta-primitives = { path = '../primitives', default-features = false }
+pallet-tx-pause = { path = '../../pallets/pallet-tx-pause', default-features = false }
```

Comment:
Do we need to import this pallet for manta-runtime for now?
I mean manta is still a shell chain.

Reply:
I think so.